.rubocop.yml

```diff
@@ -1,13 +1,21 @@
+require:
+- 'rubocop-md'
 AllCops:
   Exclude:
     - 'bin/**/*'
     - 'tmp/**/*'
+    - 'docs/web/**/*'
     - 'vendor/**/*'
     - 'gemfiles/vendor/**/*'
+    - 'CHANGELOG.md'
+    - 'README.md'
```

Comment:
Why not?

Reply:
I thought that we would get rid of ruby code in README and now it does not make sense to fix this.

Reply:
But some code in the README will remain, so
https://github.com/palkan/clowne/pull/10/commits/c0b57cfae48ad6a63ebf6ab25a038c90ccdc9d80